pyciemss/integration_utils/intervention_builder.py

```diff
@@ -102,6 +114,35 @@ def intervention_generator(
     return intervention_generator
 
 
+# def combine_intervention_templates(
```

Comment:
I do think it would be good to provide this instead of the manual lambda function expansion you have in the tests.

Reply:
I completely agree. I couldn't write a bug free one so I left it. We can hop on a call to make it happen.